srml/support/src/traits.rs

```diff
@@ -32,6 +32,20 @@ pub trait Get<T> {
 	fn get() -> T;
 }
 
+/// New trait for querying whether a type can be said to statically "contain" a value. Similar
```

Comment:
This is helpful for reviewers but with time it will get stale : )

```suggestion
/// A trait for querying whether a type can be said to statically "contain" a value. Similar
```